### PR TITLE
Updated suggestion for `JsonSerializationVisitor::addData` replacement

### PR DESCRIPTION
**Pull request by theoboldt**
When using `:visitProperty(new StaticPropertyMetadata('', 'name', 'value), null)` the 
 `\JMS\Serializer\GraphNavigator\SerializationGraphNavigator::accept()` method is called with `$data = null`, therefore `$type` will be set to `null`. Therefore, value configured in `StaticPropertyMetadata` will not be put in place (as long as it's not `null`). 

In order to make  `\JMS\Serializer\GraphNavigator\SerializationGraphNavigator::accept()` extract the correct type (via `gettype()`), the value needs to be set into `visitProperty()`. 

In conclusion, you should use `:visitProperty(new StaticPropertyMetadata('', 'name', null), 'value')`

| Q             | A
| ------------- | ---
| Bug fix?      |no
| New feature?  | no
| Doc updated   | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT

### CODE REVIEW

**Comment by goetas** [comment]
@kunicmarko20 As I remember, you made a PR about this months ago.. any comment?

**Comment by kunicmarko20** [comment]
Yes, and this is exactly the change I am using, forgot to make a PR. @theoboldt there is a code coment explain this also, can you find that one and update it too.

**Comment by kunicmarko20** [comment]
https://github.com/schmittjoh/serializer/blob/d12427b868a0f9884dbe2c0285cdaa1fa834becb/src/JsonSerializationVisitor.php#L169

**Comment by goetas** [comment]
Thanks!
